src/shared/views/Resource.tsx

```diff
@@ -20,28 +22,53 @@ interface ResourceViewProps {
 
 const ResourceViewPage: React.FunctionComponent<ResourceViewProps> = props => {
   const { match, resource, error, isFetching, fetchResource } = props;
+  const [dotGraph, setDotGraph] = React.useState<string | null>(null);
+  const [busy, setBusy] = React.useState(isFetching);
   const fetch = () => {
     fetchResource(
       match.params.org,
       match.params.project,
       match.params.resourceId
     );
   };
-  React.useEffect(
-    () => {
-      fetch();
-    },
-    [match.params.resourceId]
-  );
+  // Fetch Resource
+  React.useEffect(() => {
+    fetch();
+  }, [match.params.resourceId]);
+
+  // Once we have a legit resource, let's fetch the DOT
+  React.useEffect(() => {
+    if (resource) {
+      setBusy(true);
+      // cannot use instance method here,
+      // because sometimes we'll get 'resource'
+      // as a raw js object from the server and not an instance
+      Resource.getSelfRawAs(resource.self, ResourceGetFormat.DOT)
```

Comment:
This should probably be a redux action

Reply:
also, when was this method added? It isn't documented https://github.com/BlueBrain/nexus-sdk-js#resources-1

Reply:
updated readme: https://github.com/BlueBrain/nexus-sdk-js/pull/94